CDP4Grapher/Views/Grapher.xaml.cs

```diff
@@ -25,6 +25,7 @@
 
 namespace CDP4Grapher.Views
 {
+    using System.Windows;
```

Comment:
Why did this pop into existence all of a sudden?

Reply:
because I experimented some things ContextMenuOpening event, and missed it during my own review...